src/main/java/com/spikes2212/util/PigeonWrapper.java

```diff
@@ -80,9 +130,19 @@ public double getYaw() {
     }
 
     /**
-     * @param yaw If you don't know what yaw is, see <a href=https://letmegooglethat.com/?q=what+is+yaw>here</a>.
+     * @param yaw if you don't know what yaw is, see <a href=https://letmegooglethat.com/?q=what+is+yaw>here</a>
      */
     public void setYaw(double yaw) {
         pigeon.setYaw(yaw);
     }
+
+    public double getPitch() {
```

Comment:
the `PigeonIMU` class has methods `getYaw()`, `getPitch()` and `getRoll()` why don't you use them instead of using `getYawPitchRoll()` and retrieving the correct value?